prestoadmin/package.py

```diff
@@ -74,6 +74,10 @@ def deploy_action(local_path, rpm_action):
 
 
 def deploy(local_path=None):
+    if not os.path.isfile(local_path):
```

Comment:
You get a pretty helpful error message in put about this:
Fatal error: [hdp-kerb3-master] error: jre-8u77-linux-x64.rpm: open failed: Permission denied

That's probably more specific than what I would give